generative_ai/embedding.py

```diff
@@ -13,10 +13,10 @@
 # limitations under the License.
 
 # [START aiplatform_sdk_embedding]
-from vertexai.preview.language_models import TextEmbeddingModel
+from vertexai.language_models import TextEmbeddingModel
 
 
-def text_embedding() -> None:
+def text_embedding() -> list:
```

Comment:
nit: what kind of list is it? specific types would help, but fine to update this in a followup PR.